biopet.wdl

```diff
@@ -78,3 +78,60 @@ task SampleConfig {
         Object values = if (defined(tsvOutput) && size(tsvOutput) > 0) then read_map(tsvOutput) else { "": "" }
     }
 }
+
+task BaseCounter {
+    String? preCommand
+    String tool_jar #Should this be of type File?
+    File bam
+    File refFlat
+    String outputDir
+    String? prefix
```

Comment:
I think for this task the prefix should be a requirement, otherwise the output files are failing